.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,49 @@
+name: Bug Report
+description: |
+  Report a bug encountered while using Taichi.
+labels: 
+  - "potential bug"
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe the bug?
+      description: |
+        A clear and concise description of what the bug is, ideally within 20 words.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: How To Reproduce?
+      description: |
+        Please post a **minimal sample code** to reproduce the bug.
+        The developer team will put a higher priority on bugs that can be reproduced within 20 lines of code. If you want a prompt reply, please keep the sample code **short** and **representative**.
+      value: |
+        ```py
+        # sample code here
+        ```
+      render: markdown
+    validations:
+      required: true
+  - type: textarea
```

Comment:
To keep the template short, we could ask users to paste log in the "problem" section?

Reply:
I think we can keep the detail here to help people give log as detail as they can

Reply:
@Zheaoli Adding to my point above, I feel like it's hard to describe the problem without pasting the error msg/log? ;)

Reply:
So I add some descriptions to remind people to add logs

<img width="697" alt="CleanShot 2022-01-27 at 21 15 07@2x" src="https://user-images.githubusercontent.com/7054676/151366133-13fc965d-9de7-4db9-ba48-63c497fac10d.png">